app/src/components/Search/PokemonHit.js

```diff
@@ -56,150 +36,127 @@ function getRefinementName(value) {
 export const PokemonHit = connectCurrentRefinements(
   ({ items: refinements, pokemon }) => {
     const spriteUrl = useShiny(pokemon.artworkUrl, pokemon.spriteShinyUrl);
-    const color = COLORS_BY_TYPE[pokemon.types[0].name.toLowerCase()];
     const statRefinements = refinements.filter(refinement =>
       refinement.attribute.startsWith("stats.")
     );
 
     return (
-      <div
-        style={{
-          borderColor: lighten(0.2, saturate(0.25, color)),
-          background: lighten(0.4, color)
-        }}
-      >
+      <div className="dcard w-1/5 mt-24">
         <PokemonLink pokemonId={pokemon.id}>
-          <header>
-            <h3>
-              <Highlight tagName="mark" attribute="names.en" hit={pokemon} />
-            </h3>
-
-            <p>
-              <Highlight tagName="mark" attribute="names.fr" hit={pokemon} />{" "}
-              <Tag>
-                fr
-              </Tag>
-              {pokemon.names.ja && (
-                <>
-                  {" "}
-                  /{" "}
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+
+          <div className={`card p-6 m-2 bg-gray-900 rounded-lg`}>
```

Comment:
No need for backticks here :)

```suggestion
          <div className='card p-6 m-2 bg-gray-900 rounded-lg'>
```

Reply:
oh yes, i'm designing in the browser so, didn't really cleaned the code, hope it will be ok to clean along the tweaks!

Reply:
same goes for perfs :)